arangodb-net-standard/Serialization/ApiClientSerialization.cs

```diff
@@ -11,7 +11,7 @@ public abstract class ApiClientSerialization : IApiClientSerialization
         /// <summary>
         /// The default serialization options.
         /// </summary>
-        public virtual ApiClientSerializationOptions DefaultOptions => 
+        public ApiClientSerializationOptions DefaultOptions =
```

Comment:
I think we should keep the property read-only. I will push an update to restore the read-only behavior.